owncloudDomain/src/main/java/com/owncloud/android/domain/availableoffline/usecases/GetFilesAvailableOfflineFromEveryAccountUseCase.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * ownCloud Android client application
+ *
```

Comment:
Who did this? 👻

Reply:
Not important, but I'll add my name there, so you can blame on me 😉